test/commands/scanner/run.test.ts

```diff
@@ -883,8 +883,9 @@ describe('scanner:run', function () {
 						const violations = ctx.stdout.split('<violation');
 						// The first list item is going to be the header, so we need to pull that off.
 						violations.shift();
-						// There should be three violations. We'll just make sure that we have the right number of them.
-						expect(violations.length).to.equal(3, 'Should be 3 violations detected in the file');
+						// ApexUnitTestClassShouldHaveAsserts, FieldNamingConventions, UnusedLocalVariable, and VariableNamingConventions
```

Comment:
This change was caused in 6.23.0 by this commit https://github.com/pmd/pmd/commit/29cef7f23dc3deb8346b7210c2acb9f948edd917